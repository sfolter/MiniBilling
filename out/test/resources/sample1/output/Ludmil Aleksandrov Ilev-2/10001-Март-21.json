{
    "documentDate": "2022-08-09T09:12:42.741684600",
    "documentNumber": "10001",
    "consumer": "Ludmil Aleksandrov Ilev",
    "reference": "2",
    "totalAmount": 547.2,
    "lines": [{
        "index": 1,
        "quantity": 152,
        "lineStart": "2021-01-01T08:00:00Z",
        "lineEnd": "2021-03-25T13:40:00Z",
        "product": "gas",
        "price": 1.8,
        "priceList": 1,
        "amount": 273.6
    }]
}