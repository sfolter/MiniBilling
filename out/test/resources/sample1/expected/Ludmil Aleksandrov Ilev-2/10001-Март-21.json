{
  "documentNumber": "10001",
  "consumer": "Ludmil Aleksandrov Ilev",
  "reference": "2",
  "totalAmount": 273.6,
  "lines": [
    {
      "index": 1,
      "quantity":152,
      "lineStart": "2021-01-01T08:00:00Z",
      "lineEnd": "2021-03-25T13:40:00Z",
      "product": "gas",
      "price": 1.8,
      "priceList": 1,
      "amount": 273.6
    }
  ]
}